x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -31,7 +31,7 @@ function ofName(name: string) {
 }
 
 function isSortableByColumn(column: IndexPatternColumn) {
-  return !column.isBucketed;
+  return !column.isBucketed && column.operationType !== 'last_value';
```

Comment:
Is there a technical reason we can't sort by last value?

Reply:
Yes, it throws an error in esaggs. It is also blocked in Visualize.